### PR TITLE
api: add HttpClient()

### PR DESCRIPTION
**Pull request by arxdsilva**
Hi Huandu! 

This enables us to reuse the client, instead of just updating it.

thanks for the great lib, keep on the good work!

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -175,3 +175,8 @@ func DefaultHttpClient() HttpClient {
 func SetHttpClient(client HttpClient) {
 	defaultSession.HttpClient = client
 }
+
+// GetHttpClient gets the http client of default session.
+func GetHttpClient() HttpClient {
```
Comment (huandu) [inline api.go]: Please change the method name to `HttpClient`. We don't use "get" in Go.

**Comment by huandu** [comment]
Thanks for your contribution.

There is a tiny issue that I hope you can fix it before I merge the PR. Thanks.

**Comment by arxdsilva** [comment]
sure! thanks for your fast reply!

**Comment by arxdsilva** [comment]
I was looking at the code and there's a similar function called 'DefaultHTTPClient', do you think that it is best to forget about this change or just rename that function?

**Comment by huandu** [comment]
Oh yes. I forgot that method. 😃 

I'm closing this PR. Still thanks for your time.

**Pull request closed without merging**